spec/reek/smells/too_many_instance_variables_spec.rb

```diff
@@ -26,36 +40,36 @@ def ivars
       src = <<-EOS
         class Empty
           def ivars
-            @aa=@ab=@ac=@ad=@ae=@af=@ag=@ah=@ai=4
-            @aa=3
+           #{ivar_sequence}
+           #{ivar_sequence}
           end
         end
       EOS
       expect(src).not_to reek_of(:TooManyInstanceVariables)
     end
 
-    it 'should report 10 ivars' do
+    it 'should report excessive ivars' do
       src = <<-EOS
         class Empty
           def ivars
-            @aa=@ab=@ac=@ad=@ae=@af=@ag=@ah=@ai=@aj=4
+            #{ivar_sequence(count: too_many_ivars)}
           end
         end
       EOS
       expect(src).to reek_of(:TooManyInstanceVariables)
     end
 
-    it 'should not report 10 ivars in 2 extensions' do
+    it 'should not report for ivars in 2 extensions' do
       src = <<-EOS
         class Full
           def ivars_a
-            @aa=@ab=@ac=@ad=@ae
+            #{ivar_sequence}
           end
         end
 
         class Full
           def ivars_b
-            @af=@ag=@ah=@ai=@aj
+            #{ivar_sequence}
```

Comment:
I just noticed this changes the meaning of this test because it will use the same sequence instead of a new one.

I understand the need to make the spec not depend on the precise default value, but I feel claritiy is lost. How about adding instead a guard expectation that notices the default has changed and encourages the developer to look at these specs?